src/instance.ts

```diff
@@ -192,6 +238,234 @@ class Instance extends common.GrpcServiceObject {
     this.databases_ = new Map<string, Database>();
   }
 
+  /**
+   * Get a reference to a Backup object.
+   *
+   * @throws {Error} If any parameter is not provided.
+   *
+   * @param {string} backupId The name of the backup.
+   * @return {Backup} A Backup object.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const backup = instance.backup('my-backup');
+   */
+  backup(backupId: string): Backup {
+    if (!backupId) {
+      throw new Error('A backup ID is required to create a Backup.');
+    }
+
+    return new Backup(this, backupId);
+  }
+
+  getBackups(query?: GetBackupsRequest): Promise<GetBackupsResponse>;
+  getBackups(callback: GetBackupsCallback): void;
+  getBackups(query: GetBackupsRequest, callback: GetBackupsCallback): void;
+  /**
+   * Query object for listing backups.
+   *
+   * @typedef {object} GetBackupsRequest
+   * @property {boolean} [autoPaginate=true] Have pagination handled
+   *     automatically.
+   * @property {number} [maxApiCalls] Maximum number of API calls to make.
+   * @property {number} [maxResults] Maximum number of items to return.
+   * @property {number} [pageSize] Maximum number of results per page.
+   * @property {string} [pageToken] A previously-returned page token
+   *     representing part of the larger set of results to view.
+   */
+  /**
+   * @typedef {array} GetBackupsResponse
+   * @property {Backup[]} 0 Array of {@link Backup} instances.
+   * @property {object} 1 The full API response.
+   */
+  /**
+   * List backups on the instance.
+   *
+   * Both completed and in-progress backups are listed if no filter is supplied.
+   *
+   * @see {@link #backup}
+   *
+   * @param query query object for listing backups.
+   * @returns {Promise<GetBackupsResponse>} when resolved, contains a paged list of backups.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const [backups] = await instance.getBackups();
+   */
+  getBackups(
+    queryOrCallback?: GetBackupsRequest | GetBackupsCallback,
+    cb?: GetBackupsCallback
+  ): void | Promise<GetBackupsResponse> {
+    const callback =
+      typeof queryOrCallback === 'function' ? queryOrCallback : cb!;
+    const query =
+      typeof queryOrCallback === 'object'
+        ? queryOrCallback
+        : ({} as GetBackupsRequest);
+
+    const reqOpts = extend({}, query, {
+      parent: this.formattedName_,
+    });
+    this.request<IBackup[]>(
+      {
+        client: 'DatabaseAdminClient',
+        method: 'listBackups',
+        reqOpts,
+        gaxOpts: query,
```

Comment:
gaxOptions should be an object the user provides within the query object. Here's an example: https://github.com/googleapis/nodejs-spanner/blob/1656e4f14f0dd24f530f36ecf1ccf34b51e726fb/src/database.ts#L1133-L1142

The documentation line should be like: https://github.com/googleapis/nodejs-spanner/blob/f8ad0ece28e99dd9c6d7e985de56bd1b376f4712/src/batch-transaction.ts#L106